build-gluster-org/jobs/regression-on-demand-full-run.yml

```diff
@@ -44,13 +73,8 @@
 
     parameters:
     - string:
-        default: refs/heads/master
-        description: 'For review 12345 and patch set 6, this will be refs/changes/45/12345/6. Default: Tip of master'
-        name: GERRIT_REFSPEC
-    - string:
-        default: master
-        description: 'Name of the branch you want to build from. We usually build from master'
-        name: GERRIT_BRANCH
+        name: sha1
```

Comment:
do we need a default value for the job ? As we trigger manually those from time to time, it would be helpful to make sure it go to the master branch by default, is possible.